pkg/action/cl001/ac006/explainer.go

```diff
@@ -5,5 +5,11 @@ import (
 )
 
 func (e *Explainer) explain(ctx context.Context) (string, error) {
-	return "", nil
+	s := `
+Check if the number of pods on master node with host network set matches the number we expect from k8scloudconfig.
```

Comment:
I am not sure what `set` means here. The description was a bit hard for me to understand, even though I have an idea about what we try to assert here. 

```suggestion
Check if the number of host network pods on tenant cluster master nodes matches the number we expect from k8scloudconfig.

```